From 59d7fec7c6908604862658a3679ac44c2c3eea44 Mon Sep 17 00:00:00 2001
From: Johan Hovold <jhovold@gmail.com>
Date: Tue, 22 Mar 2011 11:12:12 +0100
Subject: USB: cdc-acm: add missing newlines to dev_dbg and dev_err
Patch-mainline: v3.0
Git-commit: 59d7fec7c6908604862658a3679ac44c2c3eea44

Add missing newline to two dev_dbg and dev_err messages.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/class/cdc-acm.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/usb/class/cdc-acm.c
+++ b/drivers/usb/class/cdc-acm.c
@@ -341,7 +341,7 @@ exit:
 	retval = usb_submit_urb(urb, GFP_ATOMIC);
 	if (retval)
 		dev_err(&urb->dev->dev, "%s - usb_submit_urb failed with "
-			"result %d", __func__, retval);
+			"result %d\n", __func__, retval);
 }
 
 /* data interface returns incoming bytes, or we got unthrottled */
@@ -355,7 +355,7 @@ static void acm_read_bulk(struct urb *ur
 	dbg("Entering acm_read_bulk with status %d", status);
 
 	if (!ACM_READY(acm)) {
-		dev_dbg(&acm->data->dev, "Aborting, acm not ready");
+		dev_dbg(&acm->data->dev, "Aborting, acm not ready\n");
 		return;
 	}
 	usb_mark_last_busy(acm->dev);
@@ -1224,7 +1224,7 @@ made_compressed_probe:
 		snd->urb = usb_alloc_urb(0, GFP_KERNEL);
 		if (snd->urb == NULL) {
 			dev_dbg(&intf->dev,
-				"out of memory (write urbs usb_alloc_urb)");
+				"out of memory (write urbs usb_alloc_urb)\n");
 			goto alloc_fail8;
 		}
 
